test/e2e/certs/create.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/zsh
+
+openssl req \
+  -new \
+  -x509 \
+  -nodes \
+  -days 99999 \
+  -subj '/CN=my-ca' \
+  -keyout testca.key \
+  -out testca.crt
+
+openssl genrsa -out testserver.key 2048
+openssl genrsa -out testclient.key 2048
+
+openssl req \
+  -new \
+  -key testserver.key \
+  -subj "/CN=e2e_receive_mtls_single_ingestor-receive-ingestor" \
```

Comment:
I believe we can use the same certs for mTLS testing in querier (and maybe in other components too). So why not have some common name for the global cert files 🤔